packages/node/src/sdk/index.ts

```diff
@@ -140,13 +140,19 @@ function _init(
   const scope = getCurrentScope();
   scope.update(options.initialScope);
 
+  if (options.spotlight && options.integrations.some(({ name }) => name !== SPOTLIGHT_INTEGRATION_NAME)) {
+    options.integrations.push(
+      spotlightIntegration({
+        sidecarUrl: typeof options.spotlight === 'string' ? options.spotlight : undefined,
+      }),
+    );
+  }
+
   const client = new NodeClient(options);
   // The client is on the current scope, from where it generally is inherited
   getCurrentScope().setClient(client);
 
-  if (isEnabled(client)) {
-    client.init();
-  }
+  client.init();
```

Comment:
I was a bit scared when I saw this but as far as I can tell we don't extend the `init` from `BaseClient` in the derived client classes. Given we guard the base client `init` implementation this should be fine.

Reply:
Event if we extend, aren't we supposed to call `super()`? I think that's the responsibility of downstream class implementors and not something we should be worried about here?